.github/workflows/release.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       contents: write
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
`v4` is the latest major version of [`actions/checkout`](https://github.com/actions/checkout/) at the time of this writing.